File: 10 - Python/Exercícios/04 - Vetores/Exercício 06.py
#Exercício - Soma Vetores

N = int(input("Quantos valores vai ter cada vetor? "))

A = [0] * N
B = [0] * N
C = [0] * N


print("Digite os valores do vetor A:")

for i in range(N):

    A[i] = int(input(""))


print("Digite os valores do vetor B:")

for i in range(N):

    B[i] = int(input(""))


print("")
print("Vetor resultante:")

for i in range(N):

    C[i] = A[i] + B[i]
    print(C[i])

File: 07 - C++/Exercícios/04 - Vetores/Exercício 06.cpp
//Exercício - Soma Vetores

#include <iostream>

using namespace std;

int main(){

    int i, N;

    cout << "Quantos valores vai ter cada vetor? ";
    cin >> N;

    int A[N], B[N], C[N];

    cout << endl << "Digite os valores do vetor A:" << endl;
    for(i = 0; i < N; i++){
        cin >> A[i];
    }

    cout << endl << "Digite os valores do vetor B:" << endl;
    for(i = 0; i < N; i++){
        cin >> B[i];
    }

    cout << endl << "Vetor resultante:" << endl;

    for(i = 0; i < N; i++){
        C[i] = A[i] + B[i];
        cout << C[i] << endl;
    }

    return 0;

}